problems/3.conditionals.js

```diff
@@ -2,25 +2,36 @@
 * @param {number} temp - The current temperature
 * @returns {string} - A message about the temperature
 */
-function temperatureMessage() {
+function temperatureMessage(temp) {
+  if (temp = -32) {
+    console.log("It's freezing")
+  } else {
+    console.log("It's alright")
+
+  }
 
 }
 
 /** Returns "It's freezing!" if the temperature is below 32, and "It's alright" otherwise. The implementation must use a ternary.
 * @param {number} temp - The current temperatur
 * @returns {string} - A message about the temperature
 */
-function temperatureMessageUsingTernary() {
-
+function temperatureMessageUsingTernary(temp) {
+  return (temp = 32 ? "It's freezing " : "It's alright")
 }
 
-
 /** Returns "You are a senior" if the age is 65 or older, "You are an adult" if the age is 18 or older, and "You are a minor" otherwise.
 * @param {number} age - The age of the user
 * @returns {string} - A message about their age
 */
-function ageMessage() {
-
+function ageMessage(age) {
+  if (age < 65) {
+    console.log("You are a senior")
+  } else if (age < 18) {
+    console.log("You are an adult")
+  } else {
+    console.log("You are a minor")
+  }
```

Comment:
See not above about console.log vs return. This problem was very close. Also would need <= less than or equal too